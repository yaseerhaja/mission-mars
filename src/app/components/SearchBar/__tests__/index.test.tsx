import { render } from "@testing-library/react";

import { SearchBar } from "..";

describe("<SearchBar  />", () => {
  it("should match snapshot", () => {
    const loadingIndicator = render(<SearchBar />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
