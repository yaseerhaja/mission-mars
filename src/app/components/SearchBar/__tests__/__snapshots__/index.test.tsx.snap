// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchBar  /> should match snapshot 1`] = `
<div
  class="MuiBox-root css-1qtyqvy"
>
  <div
    class="MuiAutocomplete-root MuiAutocomplete-hasPopupIcon css-e3x685-MuiAutocomplete-root"
  >
    <div
      class="MuiFormControl-root MuiFormControl-marginNormal MuiFormControl-fullWidth MuiTextField-root css-17vbkzs-MuiFormControl-root-MuiTextField-root"
    >
      <label
        class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
        data-shrink="false"
        for="search"
        id="search-label"
      >
        Search
      </label>
      <div
        class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-adornedEnd MuiAutocomplete-inputRoot css-154xyx0-MuiInputBase-root-MuiOutlinedInput-root"
      >
        <input
          aria-autocomplete="list"
          aria-expanded="false"
          aria-invalid="false"
          autocapitalize="none"
          autocomplete="off"
          class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputAdornedEnd MuiAutocomplete-input MuiAutocomplete-inputFocused css-nxo287-MuiInputBase-input-MuiOutlinedInput-input"
          id="search"
          role="combobox"
          spellcheck="false"
          type="text"
          value=""
        />
        <div
          class="MuiAutocomplete-endAdornment css-1q60rmi-MuiAutocomplete-endAdornment"
        >
          <button
            aria-label="Open"
            class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium MuiAutocomplete-popupIndicator css-qzbt6i-MuiButtonBase-root-MuiIconButton-root-MuiAutocomplete-popupIndicator"
            tabindex="-1"
            title="Open"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="ArrowDropDownIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M7 10l5 5 5-5z"
              />
            </svg>
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
        <fieldset
          aria-hidden="true"
          class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
        >
          <legend
            class="css-yjsfm1"
          >
            <span>
              Search
            </span>
          </legend>
        </fieldset>
      </div>
    </div>
  </div>
</div>
`;
